service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/StandardDirectionsOrderIssuedEventHandler.java

```diff
@@ -47,11 +44,7 @@
     // Needs refactored to use NotifyObject rather than Map<String, Object>
     @EventListener
     public void notifyCafcassOfIssuedSDOAndNoticeOfProceedings(StandardDirectionsOrderIssuedEvent event) {
-        String notifyTemplate = STANDARD_DIRECTION_ORDER_ISSUED_TEMPLATE;
-
-        if (featureToggleService.isSendNoticeOfProceedingsFromSdo()) {
-            notifyTemplate = SDO_AND_NOP_ISSUED_CAFCASS;
-        }
+        String notifyTemplate = SDO_AND_NOP_ISSUED_CAFCASS;
```

Comment:
do we need variable for this at all now? maybe SDO_AND_NOP_ISSUED_CAFCASS can be  used directly